.github/workflows/kind.yaml

```diff
@@ -26,11 +26,12 @@ jobs:
           make build
           IMG=flotta-operator:latest make docker-build
           kind load docker-image flotta-operator:latest
-          timeout 1m make deploy IMG=flotta-operator || retVal=$?
+          timeout 2m make deploy IMG=flotta-operator TARGET=kind || retVal=$?
```

Comment:
the condition below fails in the logs:
```
if [ $retVal -ne 124 ]; then
            echo "Make run failed";
fi
```

produces: 
```
/home/runner/work/_temp/38528907-5d0f-42d0-ac53-c99b6d373dca.sh: line 13: [: -ne: unary operator expected
```

Reply:
it doesn't cause the job to fail, so I'll merge it, since the problem was here before this change. I'll create a ticket to address it.